Message.java

```diff
@@ -0,0 +1,20 @@
+
+public class Message {
+	
+	private String toAddress;
+	private String fromAddress;
+	private String message;
+	
+	public void getTo(String toAddress) {
```

Comment:
Eventually, yes. Code isn't really part of what is required for the first part, so this is just the very basics of what I had so far. I will add more later.